dotCMS/src/main/java/com/dotcms/util/TimeMachineUtil.java

```diff
@@ -0,0 +1,22 @@
+package com.dotcms.util;
+
+import com.dotcms.api.web.HttpServletRequestThreadLocal;
+
+import javax.servlet.http.HttpSession;
+import java.util.Optional;
+
+public class TimeMachineUtil {
+
+    private TimeMachineUtil(){}
+
+    public static Optional<String> getTimeMachineDate() {
```

Comment:
doc

Reply:
shouldn't be getSession(false)

Reply:
done https://github.com/dotCMS/core/pull/18863/commits/de4603169a1241fa8698a2dc200bd71efdb03d2b